spec/pivotal-ui-react/tab/tab_spec.js

```diff
@@ -27,6 +33,18 @@ describe('Tabs', function() {
       expect('.tab-simple nav ul.nav.nav-tabs li.active').toContainText('Tab1');
       expect('.tab-simple .tab-content .tab-pane.fade.active.in').toContainText('Content1');
     });
+
+    itPropagatesAttributes('#root .tab-simple', {
+      className: 'testSimpleTabsClass',
+      id: 'testId',
+      style: {opacity: '0.5'}
+    });
+
+    describe('Tab', function() {
+      it('passes className to the tab-pane', function() {
+        expect('#root .tab-pane:nth(0)').toHaveClass('tabPaneClass');
+      });
+    });
```

Comment:
`className` being passed to the `tab-pane` is a special-ish case of className propagation. We should probably document that in the styleguide. Also in the component's readme?